js/ui/calendar/ui.calendar.js

```diff
@@ -1139,24 +1213,39 @@ const Calendar = Editor.inherit({
     },
 
     _updateViewsValue: function(value) {
-        const newValue = value ? new Date(value) : null;
+        const multiselect = this.option('multiselect');
+
+        if(multiselect) {
+            this._updateViewsOption('values', value);
+        } else {
+            const newValue = value[0] ? new Date(value[0]) : null;
```

Comment:
1. I didn't catch - y do we get `value[0]` if multiselect is disabled? intuitively it should be the other way around
2. u can see that both `this._updateViewsOption` calls are very similar - we can get rid of this `if-else`

Reply:
I used that to avoid more if-elses, so I pass array in both cases. With single select it will be [value], with multi - [...values]